typescript/src/entities/product.ts

```diff
@@ -86,6 +86,12 @@ export class Product extends Entity {
   @Product.property("allowQuotation")
   public allowQuotation?: boolean;
 
+  @Product.property("bestPrice")
+  public bestPrice?: number;
+
+  @Product.property("unitVolume")
+  public unitVolume?: number;
+
```

Comment:
'independent' is missing.

Reply:
'savedByUsers' is missing

Reply:
'savedByCompanies' is missing

Reply:
> 'independent' is missing.

I don't see that `Product` has such an attribute

Reply:
> savedByUsers
> savedByCompanies

these ones are they, they are just spelled `saved_by_users` instead, which is (unfortunately) what the backend has named them